docs/cronjob.md

```diff
@@ -0,0 +1,31 @@
+# OpenShift cronjob
+
+There is an [OpenShift template](../openshift/template.yaml) that can be used to deploy CVEjob as cronjob.
+
+When deployed, the cronjob only handles single ecosystem. So if you need to handle all supported ecosystems,
+you need to deploy it once for each ecosystem.
+
+## Requirements
+
+GitHub token needs be present in secrets of the project where the cronjob will be deployed.
```

Comment:
whose Github token?  I guess it is not developer's.

Reply:
This is not an issue on staging/prod. The token will be there already. But if from some reason developer wants to deploy the job to dev cluster, he/she will need to provide some token. Could be developer's in that case :)